src/OSPSuite.Core/Domain/ParameterIdentifications/ParameterIdentification.cs

```diff
@@ -53,7 +55,7 @@ public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager clone
 
       public virtual bool UsesSimulation(ISimulation simulation)
       {
-         return (simulation != null && _allSimulations.Contains(simulation)) || AnyOutputOfSimulationMapped(simulation);
+         return (simulation != null && _allSimulations.Contains(simulation)) || AnyOutputOfSimulationMapped(simulation) || AnyIdentificationParametersUsing(simulation);
```

Comment:
This was missing. In case the output was not available, then the method would return that it WAS NOT using the simulation and the object would not be deleted